src/components/charts/heatMap3DPlot/MultiAxisScatter.jsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import ReactEcharts from 'echarts-for-react';
+import PropTypes from 'prop-types';
+import 'echarts';
+import 'echarts-gl';
```

Comment:
When we brought in this echarts lib to do the 3D graph in Expanding Universe we discovered that the `echarts` lib is included inside the `echaerts-for-react` package.  As such we don't need to import `echarts` a second time.  Should look something like:
`import 'echarts-gl';
import ReactEcharts from 'echarts-for-react';`
I believe it turned out that order was important.